### PR TITLE
MODSOURCE-269 Support search by indicator presence

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Implement presence or absence searches for indicators

## Approach
* Add support of `is 'present/absent'` in `IndicatorBinaryOperand`. The empty indicator represents the `#` symbol in the database.

### CODE REVIEW

**Comment by Igor-Gorchakov** [comment]
Please, update news.md, put your Jira there

**Comment by Igor-Gorchakov** [comment]
Are you sure this is a correct syntax `050.ind1 is 'present'` ? Librarians may want to check the presence of indicators in a concrete subfield, not in a marc field

**Comment by psmagin** [comment]
> Are you sure this is a correct syntax `050.ind1 is 'present'` ? Librarians may want to check the presence of indicators in a concrete subfield, not in a marc field

As I know indicators are related to the full field and not to subfields as you can see on the screenshot below
image

**Comment by psmagin** [comment]
> Please, update NEWS.md, put your Jira there

NEWS updated

**Comment by Igor-Gorchakov** [comment]
> > Are you sure this is a correct syntax `050.ind1 is 'present'` ? Librarians may want to check the presence of indicators in a concrete subfield, not in a marc field
> 
> @Igor-Gorchakov, as I know indicators are related to the full field and not to subfields as you can see on the screenshot below
> image

Yes, you are right, indicators are stored per the field, just double-checked

File: mod-source-record-storage-server/src/main/java/org/folio/services/util/parser/lexeme/operand/PresenceBinaryOperand.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
  * 010.a is 'absent'            - check sub field for absence
```
Comment (Igor-Gorchakov) [inline mod-source-record-storage-server/src/main/java/org/folio/services/util/parser/lexeme/operand/PresenceBinaryOperand.java:10]: Update javadoc, add your cases, please

Comment (psmagin) [reply mod-source-record-storage-server/src/main/java/org/folio/services/util/parser/lexeme/operand/PresenceBinaryOperand.java:10]: Updated

**Review by Igor-Gorchakov — APPROVED** [review]

**Pull request merged**